{"comments": {}, "bugs": {"694189": {"comments": [{"text": "Created attachment 566675\ntestcase_delete_trace\n\nthis error when doing a DELETE on a testcase (please see attached log trace).", "count": 0, "author": "cameron@dawsoncode.com", "id": 5779040, "bug_id": 694189, "raw_text": "this error when doing a DELETE on a testcase (please see attached log trace).", "time": "2011-10-12T23:41:12Z", "creator": "cameron@dawsoncode.com", "creation_time": "2011-10-12T23:41:12Z", "is_private": false, "attachment_id": 566675, "tags": []}]}}}